datainterface/users.jsx

```diff
@@ -0,0 +1,30 @@
+const { MongoClient } = require("mongodb");
+const ObjectId = require('mongodb').ObjectId;
+
+require('dotenv').config()
+
+const uri =
+`mongodb+srv://d-team:Test12@cluster0.f4ghe7b.mongodb.net/?retryWrites=true&w=majority`
+
+
+
+const client = new MongoClient(uri);
+
+const databaseName = 'hiking_db';
+const usersCollName = 'users';
+
+module.exports = {}
+
+// GET ALL PARKING DATA
```

Comment:
I think this comment is inaccurate... unless you have a very unusual user base, made entirely of parking lots 😄

Reply:
Oops!  Removed the comment